compiler/src/iree/compiler/Dialect/LinalgExt/IR/TilingInterfaceImpl.cpp

```diff
@@ -1362,7 +1362,8 @@ LogicalResult ArgCompareOp::generateScalarImplementation(OpBuilder &b,
                                                          ValueRange ivs) {
   uint64_t reductionDim = getDimension();
   SmallVector<Value> parallelIndices;
-  for (size_t i = 0; i < ivs.size(); ++i) {
+  size_t rank = ivs.size();
+  for (size_t i = 0; i < rank; ++i) {
```

Comment:
FYI, you can also do:
```suggestion
  for (size_t i = 0, rank = ivs.size(); i < rank; ++i) {
```